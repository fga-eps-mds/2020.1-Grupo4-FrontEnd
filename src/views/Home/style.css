.vector img {
  width: 100%;
  clip-path: view-box;
}

.vector h1 {
  font-weight: 600;
  font-family: Overpass;
  font-size: 4.5vw;
  box-sizing: border-box;
  width: 50vw;
  padding-left: 10vw;
  color: var(--white);
  position: relative;
  bottom: 55vw;
}

.vector {
  max-height: 66vw;
}

.infobarShape {
  width: 100%;
}

.infoBar {
  max-height: 160vw;
}


.infoBar h1 {
  box-sizing: border-box;
  font-size: 3vw;
  font-family: Overpass;
  margin-left: 4.5vw;
  position: relative;
  bottom: 40vw;
  color: var(--title-color);
}
.infoBar h2 {
  color: var(--secondary-color);
  box-sizing: border-box;
  width: 40vw;
  max-width:  650px;
  font-size: 2vw;
  font-family: Overpass;
  padding-left: 4.5vw;
  position: relative;
  bottom: 40vw;
}

.infoBar p {
  box-sizing: border-box;
  width: 40vw;
  max-width:  650px;
  font-weight: 300;
  font-size: 2vw;
  max-width: 50%;
  font-family: Overpass;
  padding-left: 4.5vw;
  position: relative;
  bottom: 40vw;
}
.infoBar button {
  box-sizing: border-box;
  position: relative;
  bottom: 40vw;
  margin-left: 12vw;
}


.infobarImage {
  display: inline-block;
  width: 45%;
  position: relative;
  bottom: 70vw;
  margin-left: 40vw;
  padding-left: 5.5vw;
  z-index: 1;
}

.infobarImage2 {
  display: block;
  margin-left: auto;
  width: 45%;
  position: relative;
  bottom: 55vw;
  margin-right: 4.5vw;
  padding-right: 5.5vw;
  z-index: 1;
}

.infoBar2{
  max-height: 200px;
}

.infoBar2 h1 {
  box-sizing: border-box;
  font-size: 3vw;
  font-family: Overpass;
  margin-left: 4.5vw;
  position: relative;
  bottom: 25vw;
  color: var(--title-color);
}

.infoBar2 p {
  box-sizing: border-box;
  width: 40vw;
  max-width:  650px;
  font-weight: 300;
  font-size: 2vw;
  max-width: 50%;
  font-family: Overpass;
  padding-left: 4.5vw;
  position: relative;
  bottom: 25vw;
}

.infoBar2 button {
  box-sizing: border-box;
  position: relative;
  bottom: 25vw;
  margin-left: 12vw;
}
/* span {
  color: var(--secondary-color)
} */

@media(max-width: 800px) {
  .infoBar button {
    width: 80px;
    font-size: 10px;
  }
  .infoBar2 button {
    width: 80px;
    font-size: 10px;
  }
}
