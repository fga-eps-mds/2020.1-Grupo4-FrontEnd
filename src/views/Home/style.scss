
.vector {
  max-height: 66vw;

  img {
    width: 100%;
    clip-path: view-box;
  }

  h1 {
    font-weight: 600;
    font-family: Overpass, sans-serif;
    font-size: 4.5vw;
    box-sizing: border-box;
    width: 50vw;
    padding-left: 10vw;
    color: white;
    position: relative;
    bottom: 55vw;
  }
}

.infoBar {
  max-height: 1200px;

  &--shape {
    width: 100%;
  }

  &--text {
    position: relative;
    bottom: 50vw;
    max-width: 40vw;
    text-align: justify;
    margin-left: auto;
    margin-right: 5vw;
  }

  &--image {
    display: inline-block;
    bottom:76vw;
    width: 45%;
    position: relative;
  }

  h1 {
    box-sizing: border-box;
    font-size: 3vw;
    font-family: Overpass, sans-serif;
    margin-left: 4.5vw;
    color: var(--title-color);
  }

  h2 {
    color: var(--secondary-color);
    box-sizing: border-box;
    font-size: 2vw;
    font-family: Overpass, sans-serif;
    padding-left: 4.5vw;

  }

  p {
    box-sizing: border-box;
    font-weight: 300;
    font-size: 2vw;
    font-family: Overpass, sans-serif;
    padding-left: 4.5vw;

  }
  button {
    box-sizing: border-box;
    position: relative;
    margin-left: 14vw;
  }

}




.infoBar2 {
  text-align: justify;
  position: relative;

  &--image2 {
    position: relative;
    bottom: 20vw;
    width: 30%;
    text-align: justify;
    margin-left: auto;
    margin-right: 5vw;
  }
  &--text {
    font-family: Overpass, sans-serif;
    box-sizing: border-box;
    h1 {
    font-size: 3vw;
    margin-left: 4.5vw;
    color: var(--title-color);
  }
  p {
      width: 40vw;
      font-weight: 300;
      font-size: 2vw;
      max-width: 650px;
      font-family: Overpass, sans-serif;
      padding-left: 4.5vw;
    }
  }

  button {
    box-sizing: border-box;
    margin-left: 12vw;
  }

}

@media(max-width: 800px) {

  .infoBar {
    max-height: 300px;
    
    &--image {
      bottom: 80vw;
    }
  }
  .infoBar .button.small {
    line-height: 1px;
    min-width: 80px;
    width: 80px;
    font-size: 10px;
  }
  .infoBar2 .button.small {
    line-height: 1px;
    min-width: 80px;
    width: 80px;
    font-size: 10px;
  }

}

.FAQ {
  max-width: 95vw;
  margin-bottom: 10vw;
  margin-left: 2vw;
  
  h1 {
    color: var(--title-color);
    display: flex;
    box-sizing: border-box;
    font-size: 4vw;
    font-family: Overpass, sans-serif;
    margin-top: 1vw;
    justify-content: center;
    align-items: center;
    text-align: center;
  }

  h2 {
    color: var(--secondary-color);
    box-sizing: border-box;
    font-size: 2vw;
    font-family: Overpass, sans-serif;
    padding-left: 4.5vw;

  }

  p {
    color: var(--title-color);
    width: 40vw;
    font-weight: 300;
    font-size: 1.6vw;
    font-family: Overpass, sans-serif;
    padding-left: 4.5vw;
    text-align: justify;
    margin-bottom: 2vw;
  }
  .questions {
    margin-top: 5vw;
  }

  .question_left {
    position: absolute;
    
  }

  .question_right {
    position: relative;
    left: 50vw;
  }
}