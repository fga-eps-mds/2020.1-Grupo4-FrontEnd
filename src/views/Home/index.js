import React from 'react';
import FAQ from './components/FAQ';
import { InfoBar, InfoBar2 } from './components/Infobar';
import './style.scss';


function Home() {
  return (
    <div className="home">
      <div className="home__main">
        <div className="image">
          <h1>Nós encorajamos mulheres a se apaixonarem por programação</h1>
        </div>
        <div className="image2" />
      </div>
      <InfoBar />
      <InfoBar2 />
      <FAQ />
    </div>
  );
}

export default Home;
