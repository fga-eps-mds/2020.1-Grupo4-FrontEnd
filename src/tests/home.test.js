import { shallow } from 'enzyme';
import React from 'react';
import Home from '../views/Home';

it('Expect to render Home component', () => {
  expect(shallow(<Home></Home>).length).toEqual(1)
  expect(shallow(<Home></Home>)).toMatchSnapshot()
})