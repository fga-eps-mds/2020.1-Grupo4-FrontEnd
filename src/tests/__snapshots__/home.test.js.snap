// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Home component 1`] = `
<Fragment>
  <div
    className="vector"
  >
    <img
      alt="imagemFundo"
      src="vector.svg"
    />
    <h1>
      Nós encorajamos mulheres a se apaixonarem por programação
    </h1>
  </div>
  <div
    className="infoBar"
  >
    <img
      alt="descricao"
      className="infobar--shape"
      src="infoBar.svg"
    />
    <h1
      className="title"
    >
      A iniciativa Minacademy
    </h1>
    <h2>
      Minacademy é uma plataforma para incentivar mulheres a programarem. 
    </h2>
    <p>
      Na plataforma é oferecido um curso sobre Django, um framework para desenvolvimento web.
    </p>
    <Button
      small={true}
    >
      Explorar
    </Button>
    <img
      alt="curso"
      className="infobarImage"
      src="infobarImage.png"
    />
  </div>
  <div
    className="infoBar2"
  >
    <h1>
      Como funciona
    </h1>
    <p>
      Para auxiliar o curso é possível interagir com um 
      <span>
        mentor
      </span>
      , além do 
      <span>
        fórum
      </span>
       para tirar dúvidas ou fazer uma observação. 
    </p>
    <Button
      small={true}
    >
      Explorar
    </Button>
    <img
      alt="funcionamento"
      className="infobarImage2"
      src="infobarImage2.png"
    />
  </div>
</Fragment>
`;
